web/server/web_server.c

```diff
@@ -82,6 +82,8 @@ SIMPLE_PATTERN *web_allow_badges_from = NULL;
 SIMPLE_PATTERN *web_allow_mgmt_from = NULL;
 SIMPLE_PATTERN *web_allow_streaming_from = NULL;
 SIMPLE_PATTERN *web_allow_netdataconf_from = NULL;
+SIMPLE_PATTERN *web_allow_ssl_optional = NULL;
```

Comment:
The changes here are wrong. This method checks the client IP against the IP patterns in unrelated parameters. They are also called ACLs, but they're definitely not the ACLs we're defining with bind_to.

Reply:
As we talked before, I agree 100% and I changed, I also changed the macro that I created to use acl_port instead acl from webclient.